build.properties

```diff
@@ -4,6 +4,6 @@
 #
 
 # paths to executables
-npm=/usr/local/opt/node@6/bin/npm
+npm=/usr/local/bin/npm
```

Comment:
I think I'll need to revert this to the `node@10`-specific version of npm, to conform to the hsg-project setup peculiarity noted above. Testing still in progress though...